Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/social/party/PartyFragment.java

```diff
@@ -159,11 +160,19 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 this.displayEditForm();
                 return true;
             case R.id.menu_guild_leave:
-                this.apiHelper.apiService.leaveGroup(this.group.id).compose(apiHelper.configureApiCallObserver())
-                        .subscribe(group -> {
-                            getActivity().getSupportFragmentManager().beginTransaction().remove(PartyFragment.this).commit();
-                        }, throwable -> {
-                        });
+                new AlertDialog.Builder(viewPager.getContext())
+                        .setTitle("Leave Party")
```

Comment:
The String should be translatable. So instead of having it directly as a string, it should be a string resource (Like p.e. the label for the positive and negative buttons)
Same for the alert message.